bootstrap-app/src/cmr/bootstrap/system.clj

```diff
@@ -56,6 +56,83 @@
   "Required for jobs"
   (atom nil))
 
+(def things-to-cache
+  "These are all the caches contain in this system collected here to clean up the construction of
+   sys latter in this file."
+  {acl/token-imp-cache-key (acl/create-token-imp-cache)
+   kf/kms-cache-key (kf/create-kms-cache)
+   kl/kms-short-name-cache-key (kl/create-kms-short-name-cache)
+   kl/kms-umm-c-cache-key (kl/create-kms-umm-c-cache)
+   kl/kms-location-cache-key (kl/create-kms-location-cache)
+   kl/kms-measurement-cache-key (kl/create-kms-measurement-cache)
+   provider-cache/cache-key (provider-cache/create-cache)
+   common-health/health-cache-key (common-health/create-health-cache)
+   cmn-coll-metadata-cache/cache-key (cmn-coll-metadata-cache/create-cache)
+   coll-gran-acls-caches/coll-by-concept-id-cache-key
+   (coll-gran-acls-caches/create-coll-by-concept-id-cache-client)
+
+   elastic-search-index-names-cache/index-names-cache-key
+   (elastic-search-index-names-cache/create-index-cache)
+
+   humanizer-alias-cache/humanizer-alias-cache-key
+   (humanizer-alias-cache/create-cache-client)
+
+   humanizer-report-service/humanizer-report-cache-key
+   (humanizer-report-service/create-humanizer-report-cache-client)
+
+   has-gran-or-cwic-results-feature/has-granules-or-cwic-cache-key
+   (has-gran-or-cwic-results-feature/create-has-granules-or-cwic-map-cache)})
+
+(def things-to-schedule
```

Comment:
I just thought having all this in the construction of the sys was to much to read in one place.